types/src/p2p.rs

```diff
@@ -0,0 +1,63 @@
+use libp2p::Multiaddr;
+use serde::{Deserialize, Serialize};
+use serde_repr::Deserialize_repr;
+use std::collections::HashMap;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct AddrInfo {
+    #[serde(rename = "ID")]
+    pub id: PeerId,
+    #[serde(rename = "Addrs")]
+    pub addrs: Vec<Multiaddr>,
+}
```

Comment:
Use `#[serde(rename_all = "PascalCase")]` and have a rename only for `ID`

Reply:
changed